src/modules/account/components/account-deposit/account-deposit.jsx

```diff
@@ -23,6 +30,16 @@ export default class AccountDeposit extends Component {
       height: 'auto',
       width: '100%',
     }
+    const shapeShiftButton = {
+      clear: 'both',
+      float: 'left',
+      fontWeight: 'bold',
+      textTransform: 'uppercase',
+    }
+    let shapeShiftConverter = <a href="https://shapeshift.io">Use Shapeshift</a>
+    if (parseInt(window.augur.rpc.getNetworkID(), 10) === 1) {
```

Comment:
No need for window here. Add the following to import it.

```
import { augur } from 'services/augurjs'
```

Reply:
Ah, ok. The linter was complaining when I originally tried `augur.rpc.getNetworkID()`, and I wasn't sure how to resolve it. I'll add that import line.